### DISCUSSION TITLE
I cant open even if I am the admin

### DISCUSSION THREAD

**Discussion by rgestudillo** [AntlerVC/firetable#1356] [Support / Q&A] [answered]
There are no tables in this project. Sign in with an ADMIN account to create tables.

**Comment by harinij** [comment, marked as answer]
Hi @rgestudillo once you are connected to a project - you need to create tables by using the create table button. Here is docs on it. In case you are facing any specific issue it would be useful if you can share screenshot or recording so we can look this further. https://docs.rowy.io/how-to/create-table
